tests/testResolutionsClass.m

```diff
@@ -0,0 +1,283 @@
+classdef testResolutionsClass < matlab.unittest.TestCase   
+    
+    properties
+        parameters
+        resolutions
+    end
+
+    methods(TestMethodSetup)
+        function createResolutions(testCase)
```

Comment:
Given that these properties are not modified in any of the tests, the setup method should be at the class level rather than method level - i.e., "methods(TestClassSetup)" - there's no need to re-initialise the parameters and resolutions before each individual test. Of course, the resolutions class "resolution" would still need to be initialised at the method level.